test/e2e/util_test.go

```diff
@@ -545,20 +563,50 @@ func verifyInternalIngressController(t *testing.T, name types.NamespacedName, ho
 		"--retry-delay", "20",
 		"--max-time", "10",
 	}
-	clientPod := buildCurlPod("curl-"+name.Name, echoRoute.Namespace, image, address, echoRoute.Spec.Host, extraArgs...)
+	clientPodName := types.NamespacedName{echoRoute.Namespace, "curl-" + name.Name}
```

Comment:
Be explicit with fields. 

```suggestion
	clientPodName := types.NamespacedName{Namespace: echoRoute.Namespace, Name: "curl-" + name.Name}

```

Reply:
Done